### PR TITLE
[DEV-11017] Implement new `Made with Prezly` badge in public themes

### PR DESCRIPTION
**Pull request by fgyimah**

### CODE REVIEW

File: components/MadeWithPrezly/MadeWithPrezly.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { LogoPrezly } from 'icons';
+
+import styles from './MadeWithPrezly.module.css';
+
+export function MadeWithPrezly() {
+    return (
+        <a
+            // TODO: replace with new landing page URL
+            href="https://prezly.com"
```
Comment (MohammadxAli) [inline components/MadeWithPrezly/MadeWithPrezly.tsx:9]: Shouldn't this be `https://prez.ly/storytelling-platform`?

Comment (fgyimah) [reply components/MadeWithPrezly/MadeWithPrezly.tsx:9]: Right!

**Review by pestaa — CHANGES_REQUESTED** [review]
image

Let's remove the old one.

File: components/StoryStickyBar/StoryStickyBar.tsx [hunk 2]
```diff
@@ -26,7 +26,7 @@ function StoryStickyBar({ story }: PropsWithChildren<Props>) {
                     <p className="flex items-center">
                         Powered by
                         <a
-                            href="https://www.prezly.com/"
+                            href="https://prez.ly/storytelling-platform"
```
Comment (oleg-semyonov) [inline components/StoryStickyBar/StoryStickyBar.tsx:29]: I find it particularly weird on this theme, that there are effectively two "Powered by Prezly" badges on a single screen 😕 
Good to discuss with designer, but IMO this is too much and we should probably leave just one.

image

**Review by oleg-semyonov — APPROVED** [review]
Code-wise looks good, but I'm not entirely sure about the visual impact. Please see my comment.

File: components/StoryStickyBar/StoryStickyBar.tsx (hunk 2 above)
Comment (fgyimah) [reply components/StoryStickyBar/StoryStickyBar.tsx:29]: Dropped old link

**Review by pestaa — APPROVED** [review]

**Review by MohammadxAli — APPROVED** [review]

**Review by oleg-semyonov — APPROVED** [review]

**Pull request merged**